food-truck-frontend/src/pages/_app.js

```diff
@@ -22,16 +23,20 @@ const FoodTruckApp = ({Component, pageProps}) => {
   }, []);
 
   return (
-    <Provider store={store}>
-      <Head>
-        <title>Stacked Trucks</title>
-        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
-      </Head>
-
-      <FoodTruckThemeProvider>
-        <Component {...pageProps} />
-      </FoodTruckThemeProvider>
-    </Provider>
+      <Provider store={store}>
+          <Head>
+              <title>Stacked Trucks</title>
+              <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width"/>
+          </Head>
+
+
+          <MuiThemeProvider>
```

Comment:
we're gonna end up with two of these with my PR
we'll need to fix on merging them together